### PR TITLE
SOG Template

### PR DESCRIPTION
**Pull request by Lairdd1989**
**When merged this pull request will:**
- Ammends Template files for use with SOG modpack
- Add composition for mission

- [x] Generic Template with Zues, HC and Spectator
- [x]  Zeus Op Template same as above but with Zues Slots, Player Slots and Arsenal

https://github.com/Theseus-Aegis/MissionReviews/pull/36

### CODE REVIEW

**Comment by jonpas** [comment]
Moved SOG Template to normal template so I can see changes more easily.

May do 2 separate releases (file templates) or just write some documentation regarding the required changes.

**Comment by Lairdd1989** [comment]
Main thing is seeing how the overwritten CBA settings go.
We can see how things go after this Sunday

File: template/cba_settings.sqf [hunk 1]
```diff
@@ -1,8 +1,18 @@
 // TAC Settings
-force tac_apollo_enabled = true;               // Set to 'false' to disable persistence for missions where we aren't using our own gear.
+force tac_apollo_enabled = false;               // Set to 'false' to disable persistence for missions where we aren't using our own gear.
 
 // ACE Settings
 force ace_respawn_savePreDeathGear = false;   // Save and respawn with pre-set gear upon death. (NCO ONLY)
```
Comment (Mike-MF) [inline template/cba_settings.sqf:5]:
```suggestion
force ace_respawn_savePreDeathGear = true;   // Save and respawn with pre-set gear upon death. (NCO ONLY)
```

If by default with the template tac_apollo is off, this should be on.

**Review by Mike-MF — CHANGES_REQUESTED** [review]

File: template/cba_settings.sqf (hunk 1 above)
Comment (Lairdd1989) [reply template/cba_settings.sqf:5]: JJ wanted to see the other settings like this that have been changed

Comment (jonpas) [reply template/cba_settings.sqf:5]: Yeah, I would enable this for SOG template.

**Comment by Lairdd1989** [comment]
Comments from Test
- Bandages re-opened
- Keep Loadout didnt work

Difficulty
Remove markers
Chat Messages

**Comment by jonpas** [comment]
Difficulty -> https://github.com/Theseus-Aegis/Mods/pull/536

**Comment by jonpas** [comment]
> Keep Loadout didnt work

Because it was disabled: https://github.com/Theseus-Aegis/MissionReviews/pull/35/files#diff-6751af79fcc9f6eb225646baf693db1783b88778b15530ba3f57b0f5c9cee9feR5

> Bandages re-opened

All ACE Medical and Advanced Fatigue settings were missing `force`: https://github.com/Theseus-Aegis/MissionReviews/pull/35/files#diff-6751af79fcc9f6eb225646baf693db1783b88778b15530ba3f57b0f5c9cee9feR12-R14

**Comment by jonpas** [comment]
AAR didn't work either, just forgot to add it to `-serverMod` on the server.

**Comment by Lairdd1989** [comment]
On hold untill #83

**Comment by Lairdd1989** [comment]
Happy to classify this as ready to go.
However their will be the impact of #83 
Which will need applied to this.

**Review by jonpas — APPROVED** [review]

**Review by jonpas — APPROVED** [review]

**Review by Mike-MF — APPROVED** [review]

**Pull request merged**